### PR TITLE
review pr

### PR DESCRIPTION
**Pull request by Siegfriedk**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+# .idea/compiler.xml
+# .idea/jarRepositories.xml
+.idea/modules.xml
+.idea/*.iml
+# .idea/modules
+# *.iml
+# *.ipr
+
+# CMake
+cmake-build-*/
```
Comment (Siegfriedk) [inline .gitignore:48]: you could probably throw a few things out from this file

File: README.md [hunk 2]
```diff
@@ -1 +1,5 @@
 # tractus-x-release
+
```
Comment (Siegfriedk) [inline README.md:2]: Not sure why do you move it to helm-repository.md if the readme.md is then empty

File: docs/helm-repsoitory.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+accessible under URL
+
+- https://eclipse-tractusx.github.io/tractus-x-release/charts/dev for Dev branch
+- https://eclipse-tractusx.github.io/tractus-x-release/charts/stable for Stable branch
+
+## Availability
+
+### Dev Branch
+
+Dev branch will contain all released Helm Charts of any Tractus-X sub-product. In the future, only a certain number
```
Comment (Siegfriedk) [inline docs/helm-repsoitory.md:22]: I wouldn't use the future here

File: docs/helm-repsoitory.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+Stable branch will contain all Helm charts versions of Tractus-X sub-products associated with an official Tractus-X
+release. In the future, Helm charts associated with Tractus-X versions which have reached its end of lifetime, will be
+removed from the stable branch.
+
+The Helm repository for stable branch will be updated when a new Tractus-X release or a patch update is released.
+
+## Usage
+
+### Dev Branch
```
Comment (Siegfriedk) [inline docs/helm-repsoitory.md:37]: i would assume you would mean dev repository and not branch? in helm you actually add repositories to your helm backend not branches

**Comment by carslen** [comment]
Close this PR. Please follow up [here](https://github.com/eclipse-tractusx/tractus-x-release/pull/3) and [here](https://github.com/eclipse-tractusx/tractus-x-release/pull/4).

**Pull request closed without merging**